### PR TITLE
Add Curve ETH-stETH lp strategy

### PR DESCRIPTION
**Pull request by 0xkoffee**

### CODE REVIEW

File: src/strategies/curve/strategy-curve-stecrv.sol [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+    }
+
+    function harvest() public override onlyBenevolent {
+        // Anyone can harvest it at any given time.
+        // I understand the possibility of being frontrun / sandwiched
+        // But ETH is a dark forest, and I wanna see how this plays out
+        // i.e. will be be heavily frontrunned/sandwiched?
+        //      if so, a new strategy will be deployed.
+
+        uint256 guageTokens = gauge.balanceOf(address(this));
```
Comment (purperen) [inline src/strategies/curve/strategy-curve-stecrv.sol:125]: unused variable

File: src/strategies/curve/strategy-curve-stecrv.sol [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    function getName() external override pure returns (string memory) {
+        return "StrategyCurveStETH";
+    }
+
+    function getHarvestable() external view returns (uint256) {
+        return gauge.claimable_reward(address(this), address(crv));
+    }
+
+    function getHarvestableEth() external view returns (uint256) {
+        uint256 claimableCrv = gauge.claimable_reward(address(this), address(ldo));
```
Comment (purperen) [inline src/strategies/curve/strategy-curve-stecrv.sol:77]: you swapped the variable names with the addresses

**Review by purperen — CHANGES_REQUESTED** [review]
looks good.
i would just remove the unused variable guageTokens

File: src/strategies/curve/strategy-curve-stecrv.sol (hunk 1 above)
Comment (0xkoffee) [reply src/strategies/curve/strategy-curve-stecrv.sol:125]: Nice catch, thanks!

File: src/strategies/curve/strategy-curve-stecrv.sol (hunk 2 above)
Comment (0xkoffee) [reply src/strategies/curve/strategy-curve-stecrv.sol:77]: Not sure if I get it right.. Do you mean it should be `gauge.claimable_reward(address(ldo), address(this));`?

**Pull request merged**